src/App.css

```
@@ -49,3 +49,19 @@ body {
     transform: rotate(360deg);
   }
 }
+
+/* Wipes the treemap into view from top to bottom */
+.mantine-Treemap-root svg.recharts-surface {
+  animation: wipeDown 0.8s cubic-bezier(0.16, 1, 0.3, 1) forwards;
+}
+
+@keyframes wipeDown {
```

Stylelint (17.14.0)

[error] Expected keyframe name "wipeDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)